SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -1,8 +1,12 @@
 import Cocoa
 
-class AppCenterViewController : NSViewController {
+// 100 MiB.
```

Comment:
As we're not on a mobile platform 100MiB seems to be an "okay" number but given that most macOS applications are a few 100s of MIB in size, setting our DB size to 100MiB seems a bit excessive. Why not keep it the same size as the other SDKs (which are 10MiB)?

Reply:
Fix in new commit.